pkg/integration/tests/commit/unstaged.go

```diff
@@ -5,8 +5,6 @@ import (
 	. "github.com/jesseduffield/lazygit/pkg/integration/components"
 )
 
-// TODO: find out why we can't use .SelectedLine() on the staging/stagingSecondary views.
-
```

Comment:
@jesseduffield I wasn't totally sure what you wanted to use this for in this test; below are some suggestions, but if you had something else in mind, feel free to push fixups.

Reply:
I'm happy with the existing changes